.github/scripts/build_mac.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Nit: maybe move both scripts under `app` directory for clarity?

Reply:
I would say it is fine as is as they are meant to be short-lived anyway. :)

Reply:
do you mean `.github/scripts/app/build_mac.sh`?

Reply:
Yeah, same with `bootstrap_mac.sh`, but I'm fine if you prefer to keep as is :)